package.json

```diff
@@ -85,7 +86,7 @@
     "ts-loader": "^9.4.3",
     "ts-node": "^10.0.0",
     "tsconfig-paths": "4.0.0",
-    "typescript": "^4.7.4"
+    "typescript": "^4.8.4"
```

Comment:
As of NestJS 10, [TypeScript >=4.8 is required](https://docs.nestjs.com/migration-guide).

Reply:
I would say that the required dependency updates should be made in isolation (so we'd have 1 PR for updating Typescript).

Reply:
There's a default limit of 5 when it comes to PRs opened by Dependabot. We can change this for the time being so that we see all updates in one go 👍

Reply:
> I would say that the required dependency updates should be made in isolation (so we'd have 1 PR for updating Typescript).

Whilst I agree, the tests do not pass unless they are all updated together.

> There's a default limit of 5 when it comes to PRs opened by Dependabot. We can change this for the time being so that we see all updates in one go 

This PR only upgraded `@nestjs/core` and I ended up adding all the others to fix the above.

Reply:
> I would say that the required dependency updates should be made in isolation (so we'd have 1 PR for updating Typescript).

Despite their recommendation, the `@nestjs/*` packages still seem to work after reverting the TypeScript associated upgrades so I've removed them. I will create a PR with them after this is merged.